core/src/main/java/fr/sncf/osrd/railjson/schema/infra/RJSOperationalPoint.java

```diff
@@ -1,18 +1,32 @@
 package fr.sncf.osrd.railjson.schema.infra;
 
+import com.squareup.moshi.Json;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import fr.sncf.osrd.railjson.schema.common.Identified;
+import fr.sncf.osrd.railjson.schema.infra.trackranges.RJSOperationalPointPart;
+import java.util.List;
 
-@SuppressFBWarnings({"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"})
+@SuppressFBWarnings({"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD", "UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD"})
 public class RJSOperationalPoint implements Identified {
-    public String id;
+    public String name;
+    public int ci;
+    public String ch;
 
-    public RJSOperationalPoint(String id) {
-        this.id = id;
+    @Json(name = "ch_short_label")
+    public String chShortLabel;
+
+    @Json(name = "ch_long_label")
+    public String chLongLabel;
+
+    public List<RJSOperationalPointPart> parts;
+
+    public RJSOperationalPoint(String name, List<RJSOperationalPointPart> parts) {
+        this.name = name;
+        this.parts = parts;
     }
 
     @Override
     public String getID() {
-        return id;
+        return name;
```

Comment:
We shouldn't use name as `ID`, the field `id` is missing and should be use as unique identifier.
I'm not sure we need the following fields: `name`, `ci`, `ch`, `chShortLabel` and `chLongLabel`.